syft/generic/frameworks/hook/hook.py

```diff
@@ -538,8 +540,10 @@ def overloaded_native_method(self, *args, **kwargs):
                 method = getattr(new_self, method_name)
                 response = method(*new_args, **new_kwargs)
 
-                response.parents = (self.id, new_self.id)
+                if isinstance(response, crypten.mpc.MPCTensor):
+                    return response
 
+                response.parents = (self.id, new_self.id)
```

Comment:
Not sure I understand the role of those 3 lines for the moment!

Reply:
Removed them